cocotb/drivers/__init__.py

```diff
@@ -215,32 +214,29 @@ class BusDriver(Driver):
         * a name
         * an entity
 
-        Args:
-            entity (SimHandle): A handle to the simulator entity.
-            name (str or None): Name of this bus. ``None`` for a nameless bus, e.g.
-                bus-signals in an interface or a ``modport``.
-                (untested on ``struct``/``record``, but could work here as well).
-            clock (SimHandle): A handle to the clock associated with this bus.
-            array_idx (int or None, optional): Optional index when signal is an array.
-            bus_separator (str, optional): Character(s) to use as separator between bus
-                name and signal name. Defaults to '_'.
+    Args:
+        entity (SimHandle): A handle to the simulator entity.
+        name (str or None): Name of this bus. ``None`` for a nameless bus, e.g.
+            bus-signals in an interface or a ``modport``.
+            (untested on ``struct``/``record``, but could work here as well).
+        clock (SimHandle): A handle to the clock associated with this bus.
+        **kwargs (dict): kwargs is passed on to __init__ of the created .bus field.
+            See ``cocotb.Bus.__init__()`` for more information.
```

Comment:
Perhaps just ``Keyword arguments forwarded to :class:`cocotb.Bus`, see the docs for that class for more information.``

Reply:
OK